Makes the navbar similar to google

rajvaibhavdubey opened the pull request:
<!-- Add issue number here. If you do not solve the issue entirely, please change the message e.g. "Addresses #IssueNumber -->
Fixes #1398 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link
<!-- Replace "PR_NUMBER" with your pull request number. This link is generated when your PR passes the travis tests.A sample link can look like https://pr-200-fossasia-susper.surge.sh -->

#### Changes proposed in this pull request:

<!-- Changes: Add here what changes were made in this issue and if possible provide links. -->
- The search bar has been rounded
- The background of the navbar changed to white
- A seperate component has been created for the same because on the homepage the original search-bar is being used.


<!-- Demo Link: Add here the link where you changes can be seen. -->

- https://pr-1401-fossasia-susper.surge.sh

subhahu123 commented:
i there is no need to create component for that because they are same except that now bar is round 
we can just create a ng-classs where we can use round css for result search (round bar) and one for index bar ( corner ) . i find no to create a seperate component for this ..as this is not required for just create a bar round @rajvaibhavdubey

subhahu123 commented:
moreover there is  a problem with top-left corner
![screenshot-pr-1401-fossasia-susper surge sh-2019 01 11-22-07-51](https://user-images.githubusercontent.com/34541684/51047040-999ac400-15ed-11e9-9ba6-c53276b0ab83.png)
``

shreyanshdwivedi reviewed (commented):
I also don't find any specific reason for use of new component for making the `search-bar` rounded. 
Also, I'll suggest not to use white background only for search-bar. Instead use same background color for the `search-bar` as well as `search-options-field` just like `Google`.
![screenshot_2019-01-11 susper - decentral search engine](https://user-images.githubusercontent.com/25428397/51050673-9573a400-15f7-11e9-8d9f-dee2d2cdba7d.png)

rajvaibhavdubey commented:
@subhahu123 @shreyanshdwivedi I created a different component because we need two types of search-bar , similar to google(rectangular on index and rounded on results page). I could'nt find a way to do the same without creating a new component. If you know one, do tell me :sweat_smile: .
@shreyanshdwivedi Will do the changes .

subhahu123 reviewed (changes requested):
fix top left corner

subhahu123 reviewed (changes requested):
![screenshot_2019-01-22 susper - decentral search engine](https://user-images.githubusercontent.com/34541684/51530201-6f080100-1e60-11e9-8d48-223bb8253f83.png)
still not working

rajvaibhavdubey commented:
@subhahu123 I have not fixed it yet, Please wait.

rajvaibhavdubey committed:
Makes the navbar similar to googlex

- The search bar has been rounded
- The background of the navbar changed to white
- A seperate component has been created for the same.

Fixes #1398

rajvaibhavdubey commented:
@shreyanshdwivedi @subhahu123 @praveenojha33 Why is the test covergae decreasing for the components I haven't made changes to like intelligence, navbar, results.

praveenojha33 reviewed (commented):
@rajvaibhavdubey I think those components are connected with the components you made changes.

sakshee-19 commented on src/app/navbar/navbar.component.html:
```diff
@@ -1,14 +1,14 @@
 <nav
   class="top-nav navbar navbar-static-top navbar-default"
-  [style.background-color]="themeService.navbarbgColor"
+  style="background-color:white"
```
Move it to css file.

sakshee-19 commented on src/app/search-bar-results/search-bar-results.component.css:
```diff
@@ -0,0 +1,145 @@
+    }
+  }
+  
+  
```
Remove extra lines.

sakshee-19 commented on src/app/search-bar-results/search-bar-results.component.css:
```diff
@@ -0,0 +1,145 @@
+      width: 63vw;
+    }
+  }
+  
```
Add EOF.

sakshee-19 commented on src/app/search-bar-results/search-bar-results.component.html:
```diff
@@ -0,0 +1,35 @@
+      }
+    })
+  </script>
+  
```
Add EOF.

sakshee-19 reviewed (changes requested):
Please do some changes.

simsausaurabh commented:
Closing due to inactivity, please follow up with latest codebase.
